ext/opentelemetry-ext-otcollector/tests/test_otcollector_trace_exporter.py

```diff
@@ -263,6 +268,11 @@ def test_translate_to_collector(self):
             output_spans[0].links.link[0].type,
             trace_pb2.Span.Link.Type.TYPE_UNSPECIFIED,
         )
+        self.assertEqual(
+            output_spans[1].status.code,
+            trace_api.status.StatusCanonicalCode.INTERNAL.value,
+        )
+        self.assertEqual(output_spans[1].status.message, "")
```

Comment:
I think this particular test should not be included here as it's testing something implemented in a different place.